client/src/redux/sagas/index.js

```diff
@@ -37,10 +37,7 @@ import {
 } from './tickets.sagas';
 import { watchAuthenticatedUser } from './authenticated_user.sagas';
 import { watchWorkbench } from './workbench.sagas';
-import {
-  watchFetchWelcomeMessages,
-  watchSaveWelcomeMessages
-} from './welcome.sagas';
+import { watchFetchIntroMessages, watchSaveIntroMessages } from './intro.sagas';
```

Comment:
@Avila0000 it's complaining because there is no `intro.sagas.js`, it is still called `welcome.sagas.js`. You'll want to rename that file.

Reply:
File name was changed changed